defaults.json

```
@@ -5,5 +5,5 @@
 		"default": "node",
 		"node": "https://nodejs.org/dist/"
 	},
-	"bootstrap": "node/16.20.2"
+	"bootstrap": "node/18.18.2"
```

ref: https://nodejs.org/en/blog/release/v18.20.6/

```suggestion
	"bootstrap": "node/18.20.6"
```